create-release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Creates new package release in git.
+
+set -e
+set -o pipefail
+
+fail() {
+    echo "Error: $*" 1>&2
+    exit 1
+}
+
+if ! git diff --quiet; then
+    fail 'Make sure you have no uncommitted changes in your repository.'
+fi
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+if [[ "$current_branch" != "release" ]]; then
+    fail 'Current git branch should be "release".'
+fi
+
+echo "Current git commit is: $(git describe)"
+
+read -p 'Enter new version (e.g. "1.8.0"): ' version
+if [[ ! "$version" =~ ^[1-9][0-9]*\.[0-9]+\.[0-9]+$ ]]; then
+    fail 'Unexpected version format.'
+fi
+
+read -p 'Enter new release number: ' release
+if [[ ! "$release" =~ ^[1-9][0-9]*$ ]]; then
+    fail 'Unexpected release format.'
+fi
+
+sed -i 's/\(^Version:\s*\).*/\1'"$version"'/' pdc.spec
+sed -i 's/\(^Release:\s*\).*/\1'"$release"'%{?dist}/' pdc.spec
+sed -i 's/\(^VERSION = \)".*/VERSION = "'"$version"'"/' pdc/__init__.py
+
+git add pdc/__init__.py
+tito tag --keep-version
```

Comment:
If we don't need to change the logs or we don't want to write :wq, maybe can use a here document here.
tito tag --keep-version << WQ
:wq
WQ

Reply:
You're expecting user to use specific editor. This wouldn't work with e.g.

```bash
EDITOR=nano tito tag --keep-version
```

Anyway, it's probably good idea to edit the changelog.

Reply:
(Note: Just noticed that there is `--accept-auto-changelog` tito argument that skips the editing step.)

Reply:
Yes, I think most of the time, the editor should be vim. But as a ubuntu, sometime the default is gedit.

And also agree that edit log might be better, LGTM then :).

Reply:
> Yes, I think most of the time, the editor should be vim.

Yes, I concur, Vim rules! :) AFAIK, the default editor is nano (usually installed on a bare system) or something that allows any user to easily [exit the editor](https://stackoverflow.com/questions/11828270/how-to-exit-the-vim-editor).